sparse_strips/vello_sparse_tests/tests/image.rs

```diff
@@ -15,7 +15,7 @@ use vello_dev_macros::vello_test;
 
 pub(crate) fn load_image(name: &str) -> Arc<Pixmap> {
     let path = Path::new(env!("CARGO_MANIFEST_DIR")).join(format!("tests/assets/{name}.png"));
-    Arc::new(Pixmap::from_png(&std::fs::read(path).unwrap()).unwrap())
+    Arc::new(Pixmap::from_png(std::fs::File::open(path).unwrap()).unwrap())
```

Comment:
I don't know if this is a use case where we should use `BufReader`. It looks like png's docs don't suggest that, so it's probably fine. And this is only for tests anyway.

Reply:
`png` seems to use `BufReader` under the hood, but it's not documented: https://docs.rs/png/0.17.16/src/png/decoder/read_decoder.rs.html#20-23.